server/routes/record.routes.ts

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import { ChunkController } from '../controllers/chunk.controller';
+import {Chunk} from "../models/chuck.model";
+import {RecordController} from "../controllers/record.controller";
+
+const router = Router();
+
+router.get('/getAllRecords/:userId', RecordController.getAllRecordsById);//
```

Comment:
This is not following the REST API convention.
To get all the user records we simply need `router.get('/records')`. Also method name should be `getAllRecords`. 
we dont need to supply the user id, the request comes with the token already. Please look for example in the logout method `const { userId } = req.body;`